styles.css

```diff
@@ -170,17 +188,23 @@ body {
 }
 
 .work-showcase {
+  display: flex;
+  flex-direction: column;
+  gap: 24px;
   background: #fff;
   padding-bottom: 120px;
+  width: 100%;
+  margin: 0 auto;
```

Comment:
- [Optional] Your design is really interesting @TaiwoEnoch, Just a suggestion here is that your project list is a bit darker compared with the Figma design. It's a major skill to be able to translate a design into its exact replica, hence You can work on the overlay of a dark shadow on your projects to make it look like the Figma design.

### Your design 
![Screenshot 2023-01-21 011050](https://user-images.githubusercontent.com/69077061/213814783-8a317538-cb42-481a-a9c3-c8bf081c09e4.png)

### Figma design 
![Screenshot 2023-01-21 011114](https://user-images.githubusercontent.com/69077061/213814790-cd0d847e-2241-4d5e-9721-cda616667e2c.png)

Reply:
Thank you for your observation I'm grateful. 👍